syntaxes/src/racket.yaml

```diff
@@ -1054,7 +1054,7 @@ repository:
   comment-sexp:
     patterns:
       - name: comment.sexp.racket
-        match: "(?<=^|[()\[\]{}\",'`;\s])#;"
+        match: (?<=^|[()\[\]{}\",'`;\s])#;
```

Comment:
js-yaml errors out on what was originally here. Removing the quotes solves the issue.